Command/GenerateDoctrineEntityCommand.php

```diff
@@ -216,9 +218,28 @@ private function addFields(InputInterface $input, OutputInterface $output, Quest
         }
         $output->writeln('');
 
-        $fieldValidator = function ($type) use ($types) {
+        $output->write('<info>Available relations:</info> ');
+
+        $relations = array('many_to_many', 'many_to_one', 'one_to_one', 'one_to_many');
+        foreach ($relations as $i => $relation) {
+            $count += strlen($relation);
```

Comment:
What is the indended use for this? You're counting how many chars have the relationship's names and this var seems to be unused.